src/navigation.ts

```diff
@@ -1139,7 +1139,7 @@ export class Navigation {
     // Although it seems like this should never happen, the typings are wrong
     // in the base Marker class and this can therefore be null.
     if (this.markedNode) {
-      this.passiveFocusIndicator.show(this.markedNode);
+      // this.passiveFocusIndicator.show(this.markedNode);
```

Comment:
TODO: Get resolution from Rachel and/or Christopher before finalizing this solution and moving PR to review.

Reply:
This has been fixed in the latest, PTAL.